src/components/typeahead/typeahead-item.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+interface TypeaheadItemProps {
+  id: string;
+  text: string;
+  description?: string;
+  searchText?: string;
+  onSelect?: (text: string) => void;
+}
+
+function transformText(text: string, searchText?: string) {  
+  let newText = text;
+  if (searchText) {
+    const index = text.toLowerCase().indexOf(searchText.toLowerCase());
+    if (index > -1) {
+      const preText = `${text.substr(0, index)}`;
+      const postText = `${text.substr((index + searchText.length), (text.length - (index + 1)))}`;
+      const highlightText = `${text.substr(index, searchText.length)}`;
+      newText = `${preText}<strong>${highlightText}</strong>${postText}`;
```

Comment:
I dont think you need to construct markup like this.

Reply:
I'm not really sure how I can achieve text highlighting besides substr and manually injecting the font style besides this.